README.md

```diff
@@ -38,32 +38,54 @@ excluded_globs: 'frontend/*.spec.js;**/*.{png,jpg};**/!(*.module).ts'
 # - '**/!(*.module).ts'  # all TS files except module files
 ```
 
-## `root_path`
+### `root_path`
 
 The directory (and its children) that you want to visualize in the diagram, relative to the repository root.
 
 For example: `src/`
 
 Default: `''` (current directory)
 
-## `max_depth`
+### `max_depth`
 
 The maximum number of nested folders to show files within. A higher number will take longer to render.
 
 Default: 9
 
-## `commit_message`
+### `push`
+
+Whether to make a new commit with the diagram and push it to the original repository.
+
+Should be a boolean value, i.e. `true` or `false`. See `commit_message` and `branch` for how to customise the commit.
+
+Default: `false`
```

Comment:
we should probably leave this as `true`, for backwards compatibility. I know there are projects using repo-visualizer@main

Reply:
Oops, the default was indeed `true` but I miswrote it in the README! Will fix it.